src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -60,10 +63,16 @@ public class Intake extends SubsystemBase {
 
   private final int SMART_MOTION_SLOT = 0;
 
+  /*Setter for whether intake is desired deploy is true retract is false */
+  public void setDesiredDeployed(boolean desired) {
+    desiredDeployed = desired;
+  }
+
```

Comment:
Can you move this below the constructor? It's just convention but generally class definitions have members first, then constructors, then other stuff